src/tool_getparam.c

```diff
@@ -351,7 +351,8 @@ typedef enum {
   C_VERSION,
   C_WDEBUG,
   C_WRITE_OUT,
-  C_XATTR
+  C_XATTR,
+  C_TLS_SIGNED_CERT_TIMESTAMP
```

Comment:
Putting it alphabetically would be better.

Reply:
Fixed.